PrestaShop/PrestaShop
# Automatically generate no image thumbnails

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Automatically regenerates default (not found / no image available) images in front office if missing.
| Type?             | bug fix
| Category?         | FO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | See below
| UI Tests          | https://github.com/florine2623/testing_pr/actions/runs/10615886234
| Fixed issue or discussion?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/32351, Fixes https://github.com/PrestaShop/hummingbird/issues/617
| Related PRs       | 
| Sponsor company   | TRENDO s.r.o.

### How to test
- Make a product with no image
- Go to img/p/ and delete all thumbnails of not found images, keep only the one for your lang - fr.jpg for example.
- Go to FO, refresh, go back to that folder, see that all images are renegerated.
- Now delete even fr.jpg
- Go to FO, refresh, go back to that folder, see that all images are renegerated using a fallback image.
